app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,14 @@
 class PagesController < ApplicationController
   def home
+    @attendance = Attendance.new
+  end
+  def create
```

Comment:
Remember to leave a blank line between method definitions so your code is easy to read.